backend/src/packages/chaiNNer_ncnn/__init__.py

```diff
@@ -21,9 +21,9 @@
     id="chaiNNer_ncnn",
     name="NCNN",
     description=(
-        f"{general} NCNN uses Vulkan for GPU"
-        " acceleration, meaning it supports any modern GPU. Models can be converted"
-        " from PyTorch to NCNN."
+        f"{general} Models can be converted from PyTorch to NCNN.\n\nNCNN uses Vulkan"
```

Comment:
> Models can be converted from PyTorch to NCNN.

Only if ONNX is installed as well. Could you please mention that?

Reply:
Of course

Reply:
done.